korge/src/commonMain/kotlin/com/soywiz/korge/Korge.kt

```diff
@@ -587,7 +587,7 @@ object Korge {
         val quality: GameWindow.Quality? = null,
         val icon: String? = null,
         val multithreaded: Boolean? = null,
-        val main: (suspend Stage.() -> Unit)? = module.main,
+        var main: (suspend Stage.() -> Unit)? = module.main,
```

Comment:
As described below, this can be kept as immutable by using `.copy()`

Reply:
done